doc/boards.rst

```diff
@@ -207,6 +207,21 @@ ESP8285 (`datasheet <http://www.espressif.com/sites/default/files/0a-esp8285_dat
 
 Note that since ESP8285 has SPI flash memory internally connected in DOUT mode, pins 9 and 10 may be used as GPIO / I2C / PWM pins.
 
+Lifely Agrumino Lemon v4 (WT8266-S1 Module)
+-------------------------------------------
+
+Procuct page https://www.lifely.cc
+
+This Board "Lifely Agrumino Lemon" is based with WT8266-S1  core WiFi 2,4Ghz with 2MB of Flash.
+
+Power
+Micro usb power cable, Lir2450 rechargeable battery (or not rechargeable)or with JST connector  in the back board Max 6 Vin
+
+Libraries and examples
+Download the libraries from: Official Arduino Ide, our website https://www.lifely.cc or https://github.com/lifely-cc/
+
+Full pinout and PDF for setup `here https://www.lifely.cc  our libraries is OpenSource
```

Comment:
Think you've got an unterminated backtick around the "here".

Also some random spacing issues.  Maybe your editr was word-wrapping or something?  Some spaces missing (power line), some extra (this and line 215, for example).

Reply:
Of course, the fix is in the .py and not this generated file, so please fix in boards.txt.py.